src/test/java/BaseTest.java

```diff
@@ -70,6 +81,87 @@ public static void provideEmail(String email) {
+    }
+    public static void choosePlaylist(){
+<<<<<<< Updated upstream
+        WebElement choosePlaylist = driver.findElement(By.xpath("//a[contains(text(),'rock songs')]"));
+        choosePlaylist.click();
+=======
+//        WebElement playlistElement = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li.playlist.playlist:nth-child(8) > a:nth-child(1)")));
+//        playlistElement.click();
+
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)"))).click();
+>>>>>>> Stashed changes
+    }
+    public static void deletePlaylist(){
+        WebElement deletePlaylist = driver.findElement(By.cssSelector("button.del.btn-delete-playlist"));
+        deletePlaylist.click();
+<<<<<<< Updated upstream
+=======
+    }
+    public static void contextClickPlaylist(){
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playlist:nth-child(3)")));
+        WebElement playlistElement = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        action.contextClick(playlistElement).perform();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li[data-testid^='playlist-context-menu-edit']")));
+     //   wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//li[contains(text(),'Edit')]")));
+    }
+    public static void playlistNewName(){
+      //  WebElement playlistInputField = driver.findElement(By.cssSelector("input[name='name']"));
+        WebElement playlistInputField = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        playlistInputField.sendKeys(Keys.chord(Keys.CONTROL, "a", Keys.BACK_SPACE));
+        playlistInputField.sendKeys("newPlaylist");
+        playlistInputField.sendKeys(Keys.ENTER);
+    }
+    public static void doubleClickPlaylist(){
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)")));
+        WebElement playlistElement = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        action.doubleClick(playlistElement).perform();
+    }
+    public static void enterPlaylistName(){
+        WebElement playlistInputField = driver.findElement(By.cssSelector("input[name='name']"));
+        playlistInputField.sendKeys(Keys.chord(Keys.CONTROL, "a", Keys.BACK_SPACE));
+        playlistInputField.sendKeys("Cool");
+        playlistInputField.sendKeys(Keys.ENTER);
+    }
+//    public static boolean doesPlaylistExist(){
+//        WebElement playlistElement = driver.findElement(By.xpath("//a[text()="Edited Playlist Name']"));
+//        return playlistElement.isDisplayed();
+//    }
+    public static boolean IsPlaylistRenamed(){
+     //   WebElement playlistElement = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        WebElement playlistElement = driver.findElement(By.xpath("//a[contains(text(),'Cool')]"));
+        return playlistElement.isDisplayed();
+>>>>>>> Stashed changes
```

Comment:
Please remove this lines

Reply:
I really don't know why, it always have these line after push "<<<Stashed changes & Updated upstream". Probably, that's the reason why there is error. Coz before I commit & push it, it run successfully on my local machine, and no "Stashed changes & Updated upstream" lines. But now, I tried to run it again on my local machine, there are errors "at 
       "BaseTest.contextClickPlaylist(BaseTest.java:98)
	at Homework21CH.renamePlaylist(Homework21CH.java:11)"
that's the thing I am trying to resolve now. Can I commit & push Homework21CH again, after I fix it? Another issue that I encountered yesterday & today, I couldn't commit & push my "Homework22CH" hopefully, coz it says on the error, "Conflict with Homework21CH". I need help with this. Thanks for your comment.